src/pages/profile/modules/changePassword/types.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface Props {
+  avatar: unknown;
+  oldPassword: unknown;
+  newPassword: unknown;
+  newPasswordGuard: unknown;
+  savaChanges: unknown;
```

Comment:
Тут скорее всего опечатка
```suggestion
  saveChanges: unknown;
```

Reply:
Опечатка в тексте или "по смыслу"? Этот интерфейс описывает какие Компоненты ожидает внутри себя страница. Что-то вроде именованных слотов. Вместо unknown укажу типы компонентов

Reply:
опечатка в тексте. Вместо save написано sava. Но это лишь мое предположение, может действительно было задумано sava. Во всяком случае это ни в коем случае не критическая ошибка, просто наблюдение